spark-pipeline/src/main/scala/geotrellis/spark/pipeline/json/transform/Transform.scala

```diff
@@ -0,0 +1,66 @@
+package geotrellis.spark.pipeline.json.transform
+
+import geotrellis.raster.resample.{NearestNeighbor, PointResampleMethod}
+import geotrellis.raster.CellType
+import geotrellis.spark.pipeline.json._
+import geotrellis.spark.tiling.{LayoutDefinition, LayoutScheme}
+import io.circe.generic.extras.ConfiguredJsonCodec
+
+trait Transform extends PipelineExpr
```

Comment:
We already discussed this a bit. This is a reminder to think about removing these classes and just using circe encoder/decoder logic to manage serialization/deserialization